mycore-base/src/main/java/org/mycore/common/content/MCRXMLContent.java

```diff
@@ -58,7 +58,7 @@ public MCRXMLContent() {
         try {
             this.setEncoding(MCRConstants.DEFAULT_ENCODING);
         } catch (UnsupportedEncodingException e) {
-            throw new RuntimeException(e);
+            throw new IllegalArgumentException(e);
```

Comment:
This one thing still needs attention .

It's the same detail that was already improved in `MCRStringContent`, i.e. should be `throw new MCRException("MCRXMLContent encoding not supported", e);`. Probably drowned out between all the other stuff and because GitHub, for whatever reason, decided to hide the comment in the timeline.

Reply:
Strange. I think I corrected that already. Anyway, it should be now.